jsapp/js/components/modal.es6

```diff
@@ -337,7 +327,7 @@ class Modal extends React.Component {
               <div>
                 <bem.Loading>
                   <bem.Loading__inner>
-                    <i />
+                    <i className='k-spin k-icon k-icon-spinner'/>
```

Comment:
Do you think it would be better to have maybe an optional prop for `LoadingSpinner` that would hide the text? Like `no-message` attribute on the element?

Reply:
i think it would, might be useful for #3133 too

Reply:
Yes, let's move it from here to #3133 :+1: